src/lib/storage/partitioning/null_partition_schema.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <map>
+
+#include "all_type_variant.hpp"
+#include "storage/chunk.hpp"
+#include "storage/partitioning/abstract_partition_schema.hpp"
+#include "storage/proxy_chunk.hpp"
+
+namespace opossum {
+
+/*
+ * This is a null object for the PartitionSchema strategy.
+ * If there should not be any partitioning, NullPartitionSchema is used.
+ * It holds exactly one Partition where all Chunks of a Table are referenced in.
+ */
+
+class NullPartitionSchema : public AbstractPartitionSchema {
```

Comment:
NullPartitionSchema sound like it somehow handles NULL values. Consider DummyPartitionSchema or something like that.